zubhub_frontend/zubhub/src/assets/js/styles/views/project_details/projectDetailsStyles.js

```diff
@@ -189,15 +190,48 @@ const styles = theme => ({
     height: '200px',
   },
   enlargedImageDialogStyle: {
+    padding:'4px',
+  },
+  enlargedImageDialogPaperStyle: {
+    backgroundColor: 'transparent',
+    boxShadow: 'none',
+  },
+  enlargedImageDialogBackdropStyle: {
+    opacity: '0.7',
+    backgroundColor: 'black',
+  },
+  enlargedImageDialogCloseIcon: {
+    color: colors.white,
+  },
+  sliderImageStyle: {
+    alignItems: 'center',
+    width: '100%',
+  },
+  enlargedImageContainer: {
     display: 'flex',
-    justifyContent: 'center',
+    justifyContent:'center',
+    width: '100%',
+    marginTop: '12px',  
   },
   enlargedImageStyle: {
     alignSelf: 'center',
-    width: '80%',
-    maxWidth: '600px',
+    padding:'2px',
+    width: '85%',
+    maxWidth: '840px',    
     height: 'auto',
   },
+  cancelEnlargedImageBtn: {
+    backgroundColor: colors.secondary,
+    position: 'absolute',
+    right: '60px',
```

Comment:
I want to check how it looks on mobile, I am guessing its too far from right corner

Reply:
@kamthamc Yes, it is too far. I can make it `right: '6%'` which works well for all sizes, or I can use media queries. Which one should I go with?

Reply:
You can use something like clamp 

https://developer.mozilla.org/en-US/docs/Web/CSS/clamp

Reply:
I am afraid I do not understand why we should not go with 6%. It makes the button dynamically placed at the top right corner for every screen size. Should we not use relative units like percentages that are set relative to some other value? Since we do not have a preferred value here, I don't think using clamp with 3 specific values is the right solution.

Reply:
For example if you open an application in your machine, the close button won't be responsive that has to be placed at the top right with some gaps. There are some elements that can move based on responsiveness and some elements should not move/adjust based on screen sizes.